Documentation/Blazorise.Docs/Pages/News/2024-07-15-release-notes-160.razor

```diff
@@ -131,6 +131,14 @@
     Links with <Code>Target="Target.Blank"</Code> now automatically include <Code>rel="noopener noreferrer"</Code> for enhanced security and privacy. This update prevents new pages from accessing the original page, reducing phishing risks, and ensures referrer information is not sent to the target site, thereby enhancing the security and privacy of external links by default.
 </Paragraph>
 
+<Heading Size="HeadingSize.Is4">
+    Updated NumericPicker library
+</Heading>
+
+<Paragraph>
+    As an ongoing process to improve security of Blazorise dependencies, we have updated autoNumeric.js library to use the latest version.
```

Comment:
What version?
At any point in time I read this, "latest version" doesn't tell me much. As a developer I might want to go check the automeric.js release notes or something for the version that's actually being used.

Reply:
I will anyway rewrite it all before the final release. This is just for me to not forget about it.